template_feed/Microsoft.DotNet.Common.ItemTemplates/content/BuildProps/.template.config/template.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "MSBuild",
+    "props"
+  ],
+  "name": "MSBuild Directory.Build.props file",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "An empty Directory.Build.props file which can be used to specify MSBuild properties for an entire folder",
+  "tags": {
+    "type": "item"
+  },
+  "identity": "Microsoft.Standard.QuickStarts.DirectoryProps",
+  "groupIdentity": "msbuilddirectoryprops",
```

Comment:
This Group Id Seems a bit odd, compared to the pattern we have in the other templates. (same for the other one)

Reply:
Made it capitalized and used more consistent language in the latest version.